datalake-dr-connector/src/main/java/com/sequenceiq/cloudbreak/datalakedr/DatalakeDrClient.java

```diff
@@ -97,6 +97,38 @@ public DatalakeDrStatusResponse triggerRestore(String datalakeName, String backu
         }
     }
 
+    public DatalakeDrStatusResponse getBackupStatus(String datalakeName, String actorCrn) {
+        return getBackupStatusByBackupId(datalakeName, null, null, actorCrn);
+    }
+
+    public DatalakeDrStatusResponse getBackupStatus(String datalakeName, String backupId, String actorCrn) {
+        return getBackupStatusByBackupId(datalakeName, backupId, null, actorCrn);
+    }
+
+    public DatalakeDrStatusResponse getBackupStatus(String datalakeName, String backupId, String backupName, String actorCrn) {
+        if (!datalakeDrConfig.isConfigured()) {
+            return missingConnectorResponse();
+        }
+
+        checkNotNull(datalakeName);
+        checkNotNull(actorCrn);
+
+        try (ManagedChannelWrapper channelWrapper = makeWrapper()) {
+            BackupDatalakeStatusRequest.Builder builder = BackupDatalakeStatusRequest.newBuilder()
+                    .setDatalakeName(datalakeName);
+            if (!Strings.isNullOrEmpty(backupId)) {
+                builder.setBackupName(backupId);
```

Comment:
It should be builder.setBackupId(backupId);

Reply:
Good catch! Thanks, I'm fixing this.